src/torchmetrics/utilities/plot.py

```diff
@@ -181,6 +181,18 @@ def _get_col_row_split(n: int) -> Tuple[int, int]:
     return ceil(nsq), ceil(nsq)
 
 
+def _get_text_color(patch_color: tuple[float, float, float]) -> str:  # type: ignore[valid-type]
```

Comment:
```suggestion
def _get_text_color(patch_color: Tuple[float, float, float]) -> str:  # type: ignore[valid-type]
```
because we still support old versions of python we need to use the explicit types from the `typing` module